groovy/groovy.antproject/build.xml

```diff
@@ -19,7 +19,7 @@
     under the License.
 
 -->
-<project basedir="." default="netbeans" name="groovy.antproject">
+<project basedir="." default="build" name="groovy/groovy.antproject">
```

Comment:
Could you please explain the reason for changing the default target and the name?

Reply:
Read the explanation here: https://github.com/apache/incubator-netbeans/pull/719